test/helpers/utils.go

```diff
@@ -138,3 +139,39 @@ func Fail(description string, callerSkip ...int) {
 	}
 	ginkgo.Fail(description, callerSkip...)
 }
+
+// ReportDirectory creates and returns the directory path to export all report
+// commands that need to be run in the that a test has failed.
```

Comment:
Please be more careful with grammar and documentation of publicly-facing APIs. You can file a follow up ticket to fix this and other misc. grammatical issues - "need to be run in the case that a test has failed"